index.html

```diff
@@ -2,31 +2,38 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- <meta name="viewport" content="width=device-width, initial-scale=1.0"> -->
   <link rel="stylesheet" href="style.css">
-  <!-- <script src="main.js"></script> -->
+  <script src="main.js"></script>
   <title>Document</title>
 </head>
 <body>
-  <form class="calculator">
-    <label for="first-number">First Number:</label>
-    <!-- the "onkeyup" event listener passes its input's "value" to the "saveFirstNumber" function in the main.js file -->
-    <input type="number" id="first-Number" name="first-Number" placeholder="type the first number" value="" onkeyup="saveFirstNumber(this.value)">
-    <label for="second-number">Second Number:</label>
-    <input type="number" id="second-Number" name="second-Number" placeholder="type the second number" onkeyup="saveSecondNumber(this.value)">
-    <div>
-      <!-- the "onclick" event listener passes its element's "id" to the "changeOperation" function in the main.js file -->
-      <button type="button" name="add" id="addition" onclick="changeOperation(this.id)">Add</button>
-      <button type="button" name="subtract" id="subtraction" onclick="changeOperation(this.id)">Subtract</button>
-      <button type="button" name="multiply" id="multiplication">Multiply</button>
-      <button type="button" name="divide" id="division">Divide</button>
-      <button type="button" name="modulus" id="modulus">Modulus</button>
-    </div>
-    <br>
-    <!-- this "onclick" calls the "equal" function in the main.js file -->
-    <button type="button" onclick="equals()">Equals</button>
-    <button type="reset">Clear</button>
-  </form>
-  <div id="result"></div>
+  <main>
+    <div id="result">Look right here when you're done!</div>
+    <form class="calculator">
+      <div class="number-container">
+        <label for="first-number">First Number:</label>
+        <!-- the "onkeyup" event listener passes its input's "value" to the "saveFirstNumber" function in the main.js file -->
+        <!-- Note that "this" refers to the whole element, including the opening and closing tag -->
+        <input type="number" id="first-Number" name="first-Number" placeholder="type the first number" value="" onkeyup="saveFirstNumber(this.value)">
+        <label for="second-number">Second Number:</label>
+        <input type="number" id="second-Number" name="second-Number" placeholder="type the second number" onkeyup="saveSecondNumber(this.value)">
+      </div>
+      <div class="operation-container">
```

Comment:
Section could be great here! Also, do we use operation container on any other element in this document? It might be better to implement an ID selector.